cypress/integration/spalsh.spec.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef */

```

Comment:
Typo in this file name, it should be splash.spec.js and not spalsh.spec.js :)

Reply:
haha, that's what you get for working late I guess...